### ISSUE TITLE
Graphs Not Working

### ISSUE THREAD

**Issue by GarretOverstreet** [modmore/SimpleAB#35]
The analytics graphs don't appear to be working any longer.

**Comment by Mark-H** [comment]
Correct; that uses flash, which is blocked in most browsers. Needs to be completely rewritten.
